static/scripts/main.js

```diff
@@ -1,19 +1,47 @@
 
-
-// $(window).on('load', function() {
 document.addEventListener('DOMContentLoaded', function(){
 // Set the date we're counting down to
 //var countDownDate = new Date("Jan 5, 2021 15:37:25").getTime();
 // Get today's date and time
-let timer = '2:34:00'; // {{ quiz.duration }}
-let [h,m,s] = timer.split(':');
+// let timer = "{{quiz.duration}}";
+
+let h = "";
+let m = "";
+let s = "";
+// let newDateObj = new Date();
+// let oldDateObj = new Date();
+
+
+if(localStorage.getItem("secondsLeft") == null) {
+    let timer = document.getElementById("time").value;
```

Comment:
Hey @jtryan , where exactly did you set the `id='time'` in the `html` file again? I've tried setting a dummy `<h2>` just above the `timer` id but I still cannot get it to display.